x-pack/plugins/maps/server/maps_telemetry/collectors/register.ts

```diff
@@ -17,6 +19,13 @@ export function registerMapsUsageCollector(
     return;
   }
 
+  const clusterStats = {
+    min: { type: 'long' },
+    max: { type: 'long' },
+    avg: { type: 'float' },
```

Comment:
I'm still not sure "average" is a useful metric to capture moving forward since these are ultimately looked at in aggregate. Do we care to see the average of the averages for multiple clusters? We can also just kick the can down the road on this for now.

Reply:
tbh i just included it because we can, and better safe than sorry. agreed it's not super useful.